pkgs/by-name/wr/wrangler/package.nix

```
@@ -87,6 +96,12 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "--version" ];
```

```suggestion
  versionCheckProgramArg = "--version";
```
you can omit the parentheses.

---

It looks like `--version` is among the two defaults.

https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook

> versionCheckProgramArg: The argument that needs to be passed to versionCheckProgram. If undefined the hook tries first --help and then --version. Examples: version, -V, -v.

---

+1, just -v works